### ISSUE TITLE
Incorrect Status Value

### ISSUE THREAD

**Issue by cargueta001** [aarons22/homebridge-bond-v2#6]
@aarons22 It appears that during use of Siri and then the dedicated Bond iOS App, the status values of the light and fans goes haywire. They don't always sync up to should be the correct value. Thoughts?

**Comment by aarons22** [comment]
What version of the plugin are you using? I attempted to fix this in 0.2.1

**Comment by cargueta001** [comment]
@aarons22 I'm using the latest build. I also have a remote that I believe my wife uses instead of the Home app so my guess is that they aren't tracking the states properly. Any recommendations?

**Comment by aarons22** [comment]
Yeah I’m seeing this and am working on a fix. Should be out today!

**Comment by aarons22** [comment]
hey @cargueta001 - I just realized this conversation was still in `homebridge-bond-v2`. I've since taken over the original `homebridge-bond` project. Can you use that plugin instead? The latest version should resolve this issue
